src/Schema/TableEditor.php

```diff
@@ -43,17 +43,21 @@ final class TableEditor
     /** @internal Use {@link Table::editor()} or {@link Table::edit()} to create an instance */
     public function __construct()
     {
-        // @phpstan-ignore assign.propertyType (PHPStan doesn't infer the element type)
-        $this->columns = new UnqualifiedNamedObjectSet();
+        /** @var UnqualifiedNamedObjectSet<Column> $columns */
+        $columns       = new UnqualifiedNamedObjectSet();
+        $this->columns = $columns;
```

Comment:
Why do we want to complicate our code instead of just suppressing a false positive error?